### PR TITLE
2300 - Updated the font size for enhanced readability

### PR DESCRIPTION
**Pull request by stewartbuchan**
Changed text size to autoscale font size. @fonnesbeck @springcoil

### CODE REVIEW

**Comment by springcoil** [comment]
Looks good work man. Glad to see it. I or someone will delve into it

File: pymc3/plots/posteriorplot.py [hunk 1]
```diff
@@ -114,13 +121,17 @@ def get_trace_dict(tr, varnames):
         elif np.ndim(rope) == 1:
             rope = [rope] * var_num
 
+        if text_size is None:
+            text_size=figsize[0]
```
Comment (twiecki) [inline pymc3/plots/posteriorplot.py:125]: missing spaces, PEP8

File: pymc3/plots/posteriorplot.py [hunk 2]
```diff
@@ -114,13 +121,17 @@ def get_trace_dict(tr, varnames):
...
+
+
         for idx, (a, v) in enumerate(zip(np.atleast_1d(ax), trace_dict)):
             tr_values = transform(trace_dict[v])
             plot_posterior_op(tr_values, ax=a, kde_plot=kde_plot,
                               point_estimate=point_estimate, round_to=round_to,
                               alpha_level=alpha_level, ref_val=ref_val[idx],
                               rope=rope[idx], text_size=text_size, **kwargs)
-            a.set_title(v)
+            a.set_title(v,fontsize = text_size)
```
Comment (twiecki) [inline pymc3/plots/posteriorplot.py:134]: missing space after comma, no spaces around kwargs, PEP8

File: pymc3/plots/posteriorplot.py [hunk 3]
```diff
@@ -102,6 +108,7 @@ def get_trace_dict(tr, varnames):
 
         if ax is None:
             fig, ax = create_axes_grid(figsize, trace_dict)
+            figsize=fig.get_size_inches()
```
Comment (twiecki) [inline pymc3/plots/posteriorplot.py:111]: PEP8

File: pymc3/plots/posteriorplot.py [hunk 4]
```diff
@@ -90,6 +94,8 @@ def get_trace_dict(tr, varnames):
             figsize = (6, 2)
         if ax is None:
             fig, ax = plt.subplots(figsize=figsize)
+        if text_size is None:
+            text_size=figsize[0]
```
Comment (twiecki) [inline pymc3/plots/posteriorplot.py:98]: PEP8

File: pymc3/plots/posteriorplot.py [hunk 5]
```diff
@@ -58,6 +59,9 @@ def plot_posterior(trace, varnames=None, transform=identity_transform, figsize=N
     ax : matplotlib axes
 
     """
+
+    
+
```
Comment (twiecki) [inline pymc3/plots/posteriorplot.py:64]: remove empty lines

**Comment by twiecki** [comment]
Can you paste an example of what this looks like with this setting?

**Comment by twiecki** [comment]
Closing in favor of #2663, thanks for the contribution nonetheless!

**Pull request closed without merging**